x/commitment/keeper/msg_server_cancel_vest.go

```diff
@@ -87,7 +93,7 @@ func (k msgServer) CancelVest(goCtx context.Context, msg *types.MsgCancelVest) (
 		sdk.NewEvent(
 			types.EventTypeCommitmentChanged,
 			sdk.NewAttribute(types.AttributeCreator, msg.Creator),
-			sdk.NewAttribute(types.AttributeAmount, msg.Amount.String()),
+			sdk.NewAttribute(types.AttributeAmount, totalCancelled.String()),
```

Comment:
change attribute name to total cancelled, Also we can keep the msg.Amount as different attribute